tests/integration/pure_cairo_vm/cheatcodes/roll/roll_test.cairo

```diff
@@ -5,10 +5,38 @@ func test_roll(){
     local deployed_contract_address;
 
     %{
-        ids.deployed_contract_address = deploy_contract("main").contract_address
-        roll(123, ids.deployed_contract_address)
+        ids.deployed_contract_address = deploy_contract("./src/main.cairo").contract_address
+        roll(ids.deployed_contract_address, 123)
     %}
 
     assert_not_zero(deployed_contract_address);
+
+    local block_nb;
+    %{
+        ids.block_nb = call(ids.deployed_contract_address, "block_number_getter")[0]
+    %}
+
+    assert block_nb = 123;
+
+    return ();
+}
+
+func test_roll_with_invoke(){
+    alloc_locals;
+    local deployed_contract_address;
+
+    %{
+        ids.deployed_contract_address = deploy_contract("./src/main.cairo").contract_address
```

Comment:
Can we test it for depth 2? roll contract B, invoke contract A, A calls B, B asserts it's warped

Can be done in a separate PR